anti.md

```diff
@@ -1,43 +1,50 @@
 # Anti project
-
+Limiting Social Media Use
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
 
+
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
+The project would serve those that feel that social media is distracting and they spend too much time doing it, but still worry about missing out on important things that happen. These people often feel pressured to be checking social media (even subconcious pressure) when they want to be more engaged with the present. With this their experience could be less distracted.
```

Comment:
Yeah! I can totally relate. If you're interested in this project, read up on Fear of Missing Out (FOMO) effect. :+1: